### DISCUSSION TITLE
잠시후 다시 시도하시오

### DISCUSSION THREAD

**Discussion by taeheelee5** [changdoc/naver-vaccine-macro#66] [Q&A]
잘 실행되다가 각기 다른 병원2개를 창 2개에서만 띄우고 실행하고 나서 10초뒤쯤에 잠시후 다시시도 하시오가 뜨는데   

한개 병원만 해야하는건가요?? 

이렇게 뜨면 차단된건가요? 1시간 뒤에 풀리는 건가요?

**Comment by OneTop4458** [comment]
https://github.com/changdoc/naver-vaccine-macro/issues/42
